packages/tools/fluid-runner/README.md

```diff
@@ -1,10 +1,15 @@
 # @fluidframework/fluid-runner
 
-This package contains utility for running various functionality inside a Fluid Framework environment.
+This package contains utility for running various functionality inside an offline Fluid Framework environment.
```

Comment:
I think this still isn't super clear.  I think fluid-runner can be described as "It loads a Fluid container using a provided ODSP snapshot and provided code bundle" or something like that?  I'm not really sure what the "runner" bit is though since it seems like the main focus is on how it loads.